webapp/src/client.ts

```diff
@@ -18,16 +18,34 @@ import {
     STORAGE_CALLS_CLIENT_STATS_KEY,
     STORAGE_CALLS_DEFAULT_AUDIO_INPUT_KEY,
     STORAGE_CALLS_DEFAULT_AUDIO_OUTPUT_KEY,
+    STORAGE_CALLS_DEFAULT_VIDEO_INPUT_KEY,
 } from 'src/constants';
+import {type BgBlurData, getBgBlurData} from 'src/local_storage';
+import Segmenter from 'src/segmenter';
 
 export const AudioInputPermissionsError = new Error('missing audio input permissions');
 export const AudioInputMissingError = new Error('no audio input available');
+export const VideoInputPermissionsError = new Error('missing video input permissions');
+export const VideoInputMissingError = new Error('no video input available');
 export const rtcPeerErr = new Error('rtc peer error');
 export const rtcPeerTimeoutErr = new Error('timed out waiting for rtc connection');
 export const rtcPeerCloseErr = new Error('rtc peer close');
 export const insecureContextErr = new Error('insecure context');
 export const userRemovedFromChannelErr = new Error('user was removed from channel');
 export const userLeftChannelErr = new Error('user has left channel');
+export const DefaultVideoTrackOptions: MediaTrackConstraints = {
+
+    // TODO: consider exposing in user preferences
+    frameRate: {
+        ideal: 30,
+    },
+    aspectRatio: {
+        ideal: 4 / 3,
+    },
+    width: {
+        ideal: 640,
+    },
```

Comment:
This is temporary, for testing things. In the absence of user preferences, we should respect the device's default aspect ratio.

Reply:
Most likely, although this is just about the source. 

I still need to review aspect ratios in general in the UI, was waiting for some designs to understand what we want to do with them. I see a few options:

- Clipping to cover all the available space without stretching (what we do today through [`object-fit: cover`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit)).
- Show the full video content but add bands (lateral or vertical) to maintain the original ratio.
- Dynamically modify widget dimensions (either height or width) to match the original ratio.
- Stretch to cover the entire space (I wouldn't do this).